src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -475,23 +475,50 @@ object EntityClient {
 
       } yield allBitstreamInfo
 
-    override def metadataForEntity(entity: Entity): F[Seq[Elem]] =
+    override def metadataForEntity(entity: Entity): F[Seq[Node]] =
```

Comment:
Would a case class be better here rather than a flat sequence? I guess you're writing it all out to a single file but we might have bugs later if the order of this method changes and the client isn't updated. What do you think?

Reply:
Do you mean the order of the things in the `Seq`,  i.e. relying on the order staying the same could be bad since it could change and therefore, we should just return a case class with the items in it and let the client that called it, order it how it pleases?